src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -37,6 +56,17 @@ public class EraproDAOUtilsImpl implements EraproDAOUtils
 	HashMap<String,AssemblySubmissionInfo> assemblySubmissionInfocache= new HashMap<String, AssemblySubmissionInfo>();
 	HashMap<String, Entry> masterCache = new HashMap<String,Entry>();
 
+	private final StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
+
+	private final WebinAuthClientService webinAuthClientService = new WebinAuthClientService(
```

Comment:
- The WebinAuthClientService should be hidden inside the WebinCliValidator.
- The sequecetools should either pass Webin username and password to WebinCliValidator to be used by BioSample client or these could be passed e.g. using ENV variables without sequencetools knowing anything about them at all.